contents/handbook/growth/sales/new-sales.md

```diff
@@ -0,0 +1,141 @@
+
+- Product/web analytics + session replay: get tracking set up, turn on replay, privacy controls, figure out user ID, get set up insights/dashboards.
+- Feature flags + experiments: snippet, FF in the code, person ID and properties for targeting, deploy flag, run the experiment. 
+- Surveys: deploy a survey
+- Data warehouse: TBD
+
+Set yourself a reminder for halfway through their evaluation period to look at their usage then put together a commercial proposal. 
+
+**5. Security review**
+
+Most customers don't need this beyond sharing our existing documentation. This step often occurs in parallel with product evaluation.  
+
+If you need help with anything data privacy or [MSA-related](/handbook/growth/sales/contracts), ping Fraser for help. 
+
+**6. Commercial evaluation**
+
+The [Contracts](/handbook/growth/sales/contracts) page has full guidance on the nuts and bolts of how to put together a commercial proposal - we use PandaDoc. 
+
+By this point, you may have started running into some objections. These are the most common, and how to handle:
+
+- Gap in the product - introduce the customer to the relevant product engineer to build together (but first agree with product team if it’s a reasonable ask). We have found this approach works exceptionally well for our newer products. 
+- Pricing issue - understand their budget, just make sure that the deal overall isn't margin negative. You can also help them tune their usage to lower costs. We don't buy customers out of existing contracts. 
+- Performance (e.g. slow dashboards) - for Extra Large, usually get Tim involved, or he can loop in the right engineer to help. 
+- Confidence in PostHog - often [this Handbook page](/handbook/finance) is enough. For Very Large companies who need to be sold a bit more on the company vision, you can get James H involved.
+
+**7. Closed - won**
+
+Hooray! This is defined as when the contract is signed by _everyone_. 'They're about to sign' - NOT CLOSED. 'I've sent a DocuSign' - NOT CLOSED EITHER.  
+
+Once the contract is signed, it lives in PandaDoc. Next step - get them set up with [billing](/handbook/growth/sales/billing). 
+
+Now it's time to set up an onboarding plan. We will templatize this, but for now you should send them something in the first week that includes:
+
+- How to manage billing/credits
+- Set up regular calls/checkins
+  - $60k+ - every 1-2 weeks for the first 6 months, then every month
+  - $20-60k - every month for the first 3 months, then quarterly
+- Schedule training for the champion and/or additional people as needed - the more people you get successfully using PostHog, the more likely they are to retain
+
+Here is minimum checklist of things that we find customers should know how to do:
+- [Actions](/docs/data/actions)
+- [Cohorts](/docs/data/cohorts)
+- [Data taxonomy](/docs/data)
+- [Notebooks](/docs/notebooks)
+- Activity log
+- Internal and test user filtering
```

Comment:
Open to ideas on how I can help templatize this, through email, landing pages, tutorials, or other assets. 

Of these, Actions is the only one heavily featured in the onboarding flow.

Reply:
Priority 1: Actions and Cohorts, we need a primer and a deep dive on all the things you can do
Priority 2: Data taxonomy
Priority 3: Others